### ISSUE TITLE
insufficient funds notification when wallet has more than enough BAT

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#3212]
## Description 
I just saw on my personal profile that I had a BR notification. When I clicked on it I got a message about insufficient funds, but my monthly budget is 20BAT and my wallet has 64BAT. I should not be getting this message. Since my contribution date is in 2 days, am I just getting the wrong notification? Meaning, should this notification be something like 'Please review your sites, auto contribute is scheduled for 2 days from now'?

## Steps to Reproduce
Unsure of steps. My setup was personal profile, wallet is a mix of self-funded and grants. Two days before a-c, notification appeared.

## Actual result:
Got insufficient funds notification 2 days before auto contribute is scheduled.
<img width="406" alt="screen shot 2019-02-04 at 1 43 22 pm" src="https://user-images.githubusercontent.com/28145373/52229461-f9019200-2882-11e9-82f5-1114b540bcd9.png">

## Expected result:
No notification, my wallet balance is 64 BAT and my monthly budget is 20 BAT:
<img width="410" alt="screen shot 2019-02-04 at 1 43 32 pm" src="https://user-images.githubusercontent.com/28145373/52229489-0880db00-2883-11e9-87c4-e3dbd3c33c0d.png">

<img width="980" alt="screen shot 2019-02-04 at 1 44 58 pm" src="https://user-images.githubusercontent.com/28145373/52229516-1cc4d800-2883-11e9-8bd7-d52f3161c660.png">

## Reproduces how often: 

Unsure

## Brave version (brave://version info)

Brave | 0.61.6 Chromium: 72.0.3626.81 (Official Build) dev (64-bit)
-- | --
Revision | ac8b982e05014492d1bd7d317628a4f22a97ffa0-refs/branch-heads/3626@{#796}
OS | Mac OS X

### Reproducible on current release:
- Does it reproduce on brave-browser dev/beta builds? yes, experienced on Dev.

### Website problems only:  
- Does the issue resolve itself when disabling Brave Shields? n/a
- Is the issue reproducible on the latest version of Chrome? n/a

### Additional Information
cc @NejcZdovc in case there's any info I can provide before contribution is attempted.

**Comment by NejcZdovc** [comment]
@LaurenWags what do you have for you recurring tips?

**Comment by LaurenWags** [comment]
@NejcZdovc nothing. no recurring tips.

**Comment by NejcZdovc** [comment]
I can't reproduce it, but I think what could be the problem. Will create PR for it

**Comment by RudyMas** [comment]
I just want to add. I saw this same message for the 2nd time in a row now. Both on my PC and Laptop. I mean, last month I got this message, and today again. I'm using version 0.59.35 (64bit) now.

I have set both of the Brave Browsers to reward 20BAT among the sites I visit and I have 80BAT in my Brave Wallet. So, plenty of BAT available.

**Comment by NejcZdovc** [comment]
@RudyMas did the last month contribution went through?

**Comment by RudyMas** [comment]
No it didn't. Well, I didn't receive any confirmation of it in the Brave Browser about the distribution of the rewards.

The last rewards that was processed successfully, was with the old version of the browser.

**Comment by dunxd** [comment]
I also get this running Brave Version 0.59.35 Chromium: 72.0.3626.81 (Official Build) (64-bit) on MacOS.

I have 20 BAT monthly payment set up, and a balance of 65 BAT. I am able to send Tips fine.

**Comment by LaurenWags** [comment]
@dunxd - did your contribution go through even though you got this message? fwiw, mine did not:
<img width="409" alt="screen shot 2019-02-08 at 9 04 13 am" src="https://user-images.githubusercontent.com/28145373/52482971-ad9bfc00-2b80-11e9-8a68-9607a8df0c36.png">

**Comment by LaurenWags** [comment]
Verified passed with

Brave | 0.60.28 Chromium: 72.0.3626.96 (Official Build) beta(64-bit)
-- | --
Revision | 84098ee7ef8622a9defc2ef043cd8930b617b10e-refs/branch-heads/3626@{#836}
OS | Mac OS X

* Verified test plan from https://github.com/brave/brave-core/pull/1592
`"walletProperties":{"altcurrency":"BAT","probi_":"0","balance":0.0,"fee_amount":30.0,"rates":{"BCH":0.000992282271179031,"BTC":0.000033457497883914,"BTG":0.011890612023777545,"DASH":0.001510625167511867,"ETH":0.000988255951926979,"EUR":0.10662618834022236,"LTC":0.002893134272841262,"USD":0.119969158104618,"XRP":0.39677434920773338},`

**Comment by eljuno** [comment]
Users reported that it still not processed with latest 0.60.45 https://community.brave.com/t/no-monthly-rewards-or-tipping-possible/43571?u=eljuno

**Comment by mightypenguin** [comment]
I've seen this for at least 3-4 months. In my case I restored my wallet from my existing MUON version of brave to the Chromium based one. My problems only started happening after converting to the Chromium browser. (also on Mac High Sierra) I am often online behind a restrictive firewall. Not sure if that would make a difference.

**Comment by RudyMas** [comment]
Same for me. It's happening since I also restored my wallet in the Chromium based Brave browser. Although, tipping seems to be working. (I have sent a friend of mine some BAT for his website and he will tell me if he receives it this month. It had been deducted from my saldo.)

**Comment by LaurenWags** [comment]
@mightypenguin @RudyMas can you both go to brave://settings/help and confirm your Brave version for me?

**Comment by mightypenguin** [comment]
I just updated after the last payment failure notification so this probably won't help.

Version 0.60.48 Chromium: 72.0.3626.121 (Official Build) (64-bit)

**Comment by mightypenguin** [comment]
Just got another insufficient funds notice while I was on a VPN (might just be blocked?).

**Comment by RudyMas** [comment]
Version 0.60.48 Chromium: 72.0.3626.121 (Official Build) (64-bit)

**Comment by NejcZdovc** [comment]
@RudyMas do you have any recurring tips?

**Comment by RudyMas** [comment]
No. I only tipped 3 websites this month, but none recurring, because I was anticipating the monthly payment would fail again and I wanted to support those 3 this month.

**Comment by NejcZdovc** [comment]
@RudyMas would really like to solve this mystery. If you have some time please let me know and would love to connect and see if we can see why this is a problem. Please let me know on nejc@brave.com

**Comment by RudyMas** [comment]
See E-mail. :)

**Comment by LaurenWags** [comment]
#3640 logged as follow-up issue to this.

**Comment by RudyMas** [comment]
![image](https://user-images.githubusercontent.com/17342220/57184431-a6546800-6ebb-11e9-91f5-6ae2c5726ace.png)

Sorry guys, still happening. And I just recently added 50 BAT to see if this could help, but nope, still got this message.

![image](https://user-images.githubusercontent.com/17342220/57184438-d1d75280-6ebb-11e9-9ce7-447e3aa684d5.png)

![image](https://user-images.githubusercontent.com/17342220/57184443-e61b4f80-6ebb-11e9-935b-dcc517d3a7cf.png)

As you can see. 140 BAT in the wallet, and a contribution set voor 20 BAT monthly. I guess there should be enough in my wallet to cover this. :)

Oh, btw, I'm at version:
![image](https://user-images.githubusercontent.com/17342220/57184448-1ebb2900-6ebc-11e9-930d-a8d0af1a5048.png)

**Comment by RudyMas** [comment]
OK, a contribution went out today, but not the 20 BAT I had set, but only 5 BAT.

Just wanted to give an update.

**Comment by LaurenWags** [comment]
@RudyMas glad a contribution went out for you. In regards to only contributing 5 BAT instead of the 20 you had budgeted, Brave only contributes to verified publishers. So, if you go to brave://rewards page, do you see something like this on the right hand panel? (yours will display a different value for BAT of course):
<img width="397" alt="Screen Shot 2019-05-08 at 5 20 27 PM" src="https://user-images.githubusercontent.com/28145373/57409185-a8694000-71b5-11e9-8735-854a81c66c70.png">

**Comment by RudyMas** [comment]
Yes. It does. It has about 60 BAT designated.

**Comment by LaurenWags** [comment]
Thanks @RudyMas - sounds like your auto contribute is occurring, but there are most likely a mix of verified and unverified sites in your auto contribute table. Once we have the monthly statement in place (you can follow progress on that here: https://github.com/brave/brave-browser/issues/930), you'll have more information about your monthly auto contributions.

**Comment by RudyMas** [comment]
Thank you for the information! The monthly statement will probably also help me to keep enough BAT in my wallet to cover it all. That's maybe what went wrong in the past few months.

**Comment by Mikeureko** [comment]
Just wanted to chime in and say that I'm getting this same issue. Last month on the 5th (my payment date, not sure if it's universal or not) I was supposed to get 6.5 BAT or so for ads, and it didn't show up in my wallet. I now have 13.7 BAT pending for ads, I'll update if it goes through this time. 

I do not have any auto-contributions set at all and haven't tried tipping anyone yet. The only BAT in my wallet is the 13.1 BAT grant I got last month or so. So I should have a total of 26.8, but again, the ads BAT didn't go through on June 5th.

**Comment by Mikeureko** [comment]
Update: I ended up getting the BAT transfer this time. Not sure why the first time didn't work. It's also worth mentioning I got the same "insufficient funds" notification on my Android app, which obviously doesn't even have ads functionality yet, so I don't see how I'd get points on it anyway. 

Side note: Any plans to connect the mobile and desktop wallets to be one and the same?
